src/v0/destinations/ga4/transform.js

```diff
@@ -325,6 +326,10 @@ const responseBuilder = (message, { Config }) => {
   }
 
   removeReservedParameterPrefixNames(payload.params);
+  const integrationsObj = getIntegrationsObj(message, "ga4");
```

Comment:
Should it be `ga4` or `Google Analytics 4` ?
https://github.com/rudderlabs/rudder-sdk-js/pull/805/files#diff-5d49e897e76914916677a00e9dc67256d66a6c692687aa1dc07c3dbd2763842aR260

Reply:
@krishna2020 we need to keep the same name here which we kept in `destinationConicalNames.js` file. so there the name is ga4 and to match that we kept here ga4. ideally it shouldn't make much difference because we are covering all ga4 names in `destinationConicalNames.js` file.